src/scripts/lexmatch-sssom-compare.py

```diff
@@ -54,6 +60,28 @@ def main(verbose: int, quiet: bool):
         logger.setLevel(level=logging.ERROR)
 
 
+@main.command("combine_unmapped_lex_exacts")
+def combine_unmapped_lex_exacts() -> None:
+    file_list = [file for file in listdir(LEXMATCH_DIR) if file.endswith("_lex_exact.tsv")]
+    # Initialize an empty DataFrame to store the merged data
+    merged_lex_exact_df = pd.DataFrame()
+
+    # Concatenate all the files into a single DataFrame
+    for file in file_list:
+        file_path = join(LEXMATCH_DIR, file)
+        df = pd.read_csv(file_path, delimiter="\t")
+        merged_lex_exact_df = pd.concat([merged_lex_exact_df, df])
```

Comment:
I don't know if it is any more performant (not that it really matters), but I tend to do `for file in file_list`, and then for each `file`, I read the `df` and append to a `df_list`. Then after the `for` loop, I do a `merged_df = pd.concat(df_list)`.

Reply:
I dont think performance matters here much, but I agree @joeflack4 I think collecting in list and then concatenation all at once is cleaner. No need to block PR for this, just good to keep in mind.